src/js/tech/flash.js

```diff
@@ -101,7 +95,8 @@ class Flash extends Tech {
     // use stageclick events triggered from inside the SWF instead
     player.on('stageclick', player.reportUserActivity);
 
-    this.el_ = Flash.embed(options['swf'], placeHolder, flashVars, params, attributes);
+    this.el_ = Flash.embed(options['swf'], flashVars, params, attributes);
+    this.el_['tech'] = this;
```

Comment:
I agree it's ugly. I was going for consistency but I'll switch them while going through the tech files.